package.json

```diff
@@ -14,6 +14,9 @@
     "processor"
   ],
   "main": "lib",
+  "files": [
+    "lib"
+  ],
```

Comment:
You can use `.npmignore` instead of this to determine what's published to the npm registry. This way, you can include important files like the readme in your bundle.